test/test_helper.rb

```diff
@@ -1,10 +1,8 @@
 require 'simplecov'
 require 'coveralls'
 require 'minitest/autorun'
-require 'minitest/reporters'
 require 'minitest/spec'
 
-Minitest::Reporters.use!(Minitest::Reporters::SpecReporter.new)
```

Comment:
Had to remove minitest-reporter because it's built into minitest 5.x versions and no longer compatible as a separate gem. (From what I was able to tell.)